README.md

```diff
@@ -26,7 +26,34 @@ instructions for `Users` or `Developers` on how to run the model.
 
 As a user, you don't need to have a MATLAB license to run the STEMMUS-SCOPE model. The workflow is executed using python and MATLAB Runtime on a Unix-like system.
 
-### Installations 
+As the STEMMUS-Scope executable only supports Unix-like systems, Windows users cannot run STEMMUS-SCOPE natively.
```

Comment:
```suggestion
As the STEMMUS-SCOPE executable only supports Unix-like systems, Windows users cannot run STEMMUS-SCOPE natively.
```

Reply:
Can we add the link to this [MATLAB documentation](https://nl.mathworks.com/help/compiler/standalone-applications.html) where the reason is explained? More specifically see the line `Standalone applications run on Windows®, Linux®, and macOS corresponding to the platform on which they are generated.`

Reply:
It is more complex than just that. If we had a matlab license I could just compile the model on my machine whenever a new STEMMUS-SCOPE version is released. I think that keeping it simple and concise here might be better.

Reply:
Besides matlab license, creating one executable file per platform is due to the fact that MATLAB does not support cross-platform compilers. Adding the documentation link provides more information for the maintainers of the codes in the future. :) perhaps we can add the link to the readme of [STEMMUS_SCOPE/exe](https://github.com/EcoExtreML/STEMMUS_SCOPE/tree/main/exe) later, see issue https://github.com/EcoExtreML/STEMMUS_SCOPE/issues/28